.github/workflows/lint-server.yml

```
@@ -16,7 +19,24 @@ jobs:
         with:
           go-version: 1.18.1
       - uses: actions/checkout@v3
+        with:
+          path: "focalboard"
+      - id: "mattermostServer"
+        uses: actions/checkout@v3
+        continue-on-error: true
+        with:
+          repository: "mattermost/mattermost-server"
```

```suggestion
          repository: "mattermost/mattermost-server"
          fetch-depth: "20" 
```

By default actions/checkout@v3 does a `semi-shallow clone` with depth:1 to improve cloning performance. 
Let's bump that to a reasonable higher (20) to be on the safe side.